aleph/tests/util.py

```diff
@@ -223,7 +224,7 @@ def setUp(self):
         if not hasattr(SETTINGS, "_global_test_state"):
             SETTINGS._global_test_state = True
             destroy_db()
-            db.create_all()
+            flask_migrate.upgrade()
```

Comment:
Previously, we created the DB schema based on the model classes. I know that some people use Alembic in a way where the model classes are the only source of truth and contain all information required to generate the schema, and then auto-generate alembic revisions based on the model classes. We don’t seem to do that though (at least not consistently), as many indexes etc. are only defined in revision files, so we ended up with inconsistent DB schemata in test and dev/prod environemnts.

Not sure if there is any reason not to use the actual migrations (except for maybe test performance and historic reason)?

I tested if this has an effect on how long it takes to execute tests but couldn’t find a significant difference.